onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -4610,13 +4610,24 @@ TEST_F(GraphTransformationTests, LayerNormWithCastFusionTest_5) {
     auto* cast_out_2 = builder.MakeIntermediate();
     auto* mul_out = builder.MakeIntermediate();
     auto* add_out_2 = builder.MakeOutput();
+    auto opset = builder.DomainToVersionMap().find("")->second;
+    onnxruntime::NodeArg* axes = nullptr;
 
-    builder.AddNode("ReduceMean", {data_arg}, {reduce_mean_out_1}).AddAttribute("axes", std::vector<int64_t>{-1});
+    if (opset >= 18) {
+      axes = builder.MakeInitializer<int64_t>({}, {-1});
```

Comment:
I missed it but it did not complain either. Most of the unit test related to the optimizers do not check that the optimized model does produce the same outputs as the original model. I was wondering why it would not fail and I left one snippet of code extracting the first parameter. With opset 18, it is keepdims. Maybe I should extend `TestGraphTransformer` to automatically check both graphs on the same output.